tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -534,6 +534,13 @@ def from_tensors(tensors):
     >>> list(dataset.as_numpy_iterator())
     [(array([1, 2, 3], dtype=int32), b'A')]
 
+    >>> # `from_tensors` adds one more dimension to the shape
```

Comment:
`from_tensors` doesn't change the shape of its input. The shape of
```[ tf.random_uniform([2, 3]), tf.random_uniform([2, 3]) ]```
 is `(2, 2, 3)`. If you pass a single `tf.random_uniform([2, 3])` instead of a list, the shape will be `(2, 3)`:

```
>>> dataset = tf.data.Dataset.from_tensors(tf.random_uniform([2, 3]))
>>> dataset.output_shapes
(2, 3)
```

I think what we might be missing here is explanation for when a user would want to use `from_tensors` as opposed to `from_tensor_slices`. The main use cases I can think of for using `from_tensors` are testing with a single example and creating artificial data, e.g. with `from_tensor_slices(example).repeat(1000)`.

Reply:
Done